include/grpcpp/impl/codegen/call.h

```diff
@@ -630,7 +631,11 @@ class CallOpSet : public CallOpSetInterface,
   // object, not across objects.
   CallOpSet(const CallOpSet& other)
       : cq_tag_(this), return_tag_(this), call_(other.call_) {}
+
   CallOpSet& operator=(const CallOpSet& other) {
+    static_assert(std::is_trivially_destructible<decltype(*this)>::value,
```

Comment:
The Interception API stores an implementation of InterceptorBatchMethods here which would make it non trivially destructibe I'm guessing

Reply:
Ok, so this will be complicated since these are getting arena allocated in the callback API. We'll have to explicitly call the destructor or delete in the callback API then.